recipes/random/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from conans import ConanFile, tools
+
+class RandomConan(ConanFile):
+    name = "random"
```

Comment:
I would prefer renaming this recipe to "effolkronium-random".

Random is a very generic name, probably there are more libraries using the same name.

VCPKG offers this library, and there they are using "effolkronium-random":

https://github.com/microsoft/vcpkg/tree/master/ports/effolkronium-random